bcp/client/src/webpack/helper/map-display.js

```diff
@@ -106,15 +107,42 @@ export function mToFeet(m) {
 export function getAssetOperatorDetails(asset) {
     const details = {};
     details.photo = asset.profilePhotoSmall ? getAssetProfilePhoto(asset) : '';
-    if (asset.operator.name || asset.operator.registeredName || asset.operator.operationalContact) {
-        details.name = cleanOperatorName(asset.operator.name ? asset.operator.name : asset.operator.registeredName);
-        details.contact =
-            asset.operator.operationalContact ? safeString(asset.operator.operationalContact) : '';
-        return require('./templates/assetOperatorDetails.hbs')(details);
+    const warningSvg = warning;
+    if (Array.isArray(asset.operator)) {
+        const aircraftData = {operator: asset.operator};
+        const operator = [];
+        for (let i = 0; i < asset.operator.length; i++) {
+            const operatorDetails = displayOperatorDetails(asset.operator[i]);
+            operator.push({index:i+1,...operatorDetails});
+        }
+        return require('./templates/assetTabs.hbs')({svg:warningSvg,operator});
+    } else {
+        const {operator} = asset;
+        const operatorDetails =displayOperatorDetails(operator);
+        return require('./templates/assetOperatorDetails.hbs')(operatorDetails);
     }
     return details.photo;
 }
 
+const displayOperatorDetails = (operator) => {
+    if (operator.name || operator.registeredName || operator.operationalContact) {
+        const name = cleanOperatorName(operator.name ? operator.name : operator.registeredName);
+        const contact =
+            operator.operationalContact ? safeString(operator.operationalContact) : '';
+        return {name, contact};
+    }
+};
+
+export function showOperatorTab(tabsId, tabId, liId) {
+    $('#' + tabsId + '-ul>li.selected').removeClass('selected');
```

Comment:
Would be good to not use jquery where there are vanilla js alternatives 
https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector
https://developer.mozilla.org/en-US/docs/Web/API/Element/classList